docs/tutorials/integrations/nginx.md

```diff
@@ -0,0 +1,374 @@
+their API key to the Conjur server. The Conjur server validates that the
+identity is authentic, then checks that the provided ID is authorized to
+[retrieve the secret][secret-get]. Assuming this check passes, the Conjur server
+returns the secret value to the client.
+
+However, this flow would be vulnerable in two ways. Suppose I were to
+impersonate the Conjur server and listen with my own illegitimate server on
+`http://conjur.local`. Then when your client goes to fetch a secret, I can take
+the API key you send and impersonate you to the real Conjur server. Now I
+control your identity and can learn your secrets without you finding out. This
+is called a **man in the middle attack**.
+
+Even if I'm not able to impersonate the Conjur server, I could still learn your
+secrets by joining your network and listening for traffic coming and going from
+the Conjur server. This is called **passive surveillance**.
+
+### TLS defeats passive and active attacks
+
+Transport Layer Security allows your client to verify that it's talking to the
+real Conjur server, and it uses standard secure technology to encrypt your
+secrets in transit. This means:
+
+* Your Conjur server will be `https:` instead of `http:`, just like a secure
+  website
+* Because the client knows it's talking to the real server, the "man in the
+  middle" will be exposed and the client won't leak any secret information
+* Since the traffic to and from Conjur is scrambled using secure encryption,
+  passive listeners on your network can't learn anything about the contents of
+  your secrets
+
+For these reasons, it is crucial to set up TLS correctly when you deploy Conjur.
+
+### Protect Your Secrets
+
+Do not leave this to chance: even a small flaw can totally compromise your
+secrets. Setting up Conjur and TLS without the appropriate expertise is like
+packing your own parachute and jumping out of a plane.
+
+That doesn't mean you should close the tab and walk away. It means you should get
+in touch with us and your own security team so we can ensure that you can employ
+Conjur successfully.
+
+[authn]: https://www.conjur.org/api.html#authentication-authenticate-post
+[secret-get]: https://www.conjur.org/api.html#secrets-retrieve-a-secret-get
+
+## Using NGINX To Proxy Traffic With TLS To Conjur
+
+[NGINX][NGINX] is a web proxy that's Free Software and easy to configure. This
+tutorial will show you how to use Docker to install Conjur and NGINX and
+configure them to provide Conjur with TLS.
+
+[NGINX]: https://www.nginx.com
+
+### Prerequisites
+
+This tutorial requires Docker and a terminal application. Prepare by following
+the prerequisite instructions found on [Install Conjur][prerequisites].
+
+[prerequisites]: /get-started/install-conjur.html#prerequisites
+
+Additionally, you will need the tutorial files from the Conjur source code
+repository. Here's how you get them:
+
+1. Install Git: https://git-scm.com/book/en/v2/Getting-Started-Installing-Git
```

Comment:
Can you make "Install Git" a hyperlink instead of including the URL in the doc?

Reply:
yep done